### DISCUSSION TITLE
SDL_HINT_VIDEO_MINIMIZE_ON_FOCUS_LOSS=0 not working

### DISCUSSION THREAD

**Discussion by taep96** [ppy/osu#24332] [Q&A] [answered]
### Type

Game behaviour

### Bug description

By default osu minimizes itself when focus is lost, I've tried disabling that with `SDL_HINT_VIDEO_MINIMIZE_ON_FOCUS_LOSS=0` and it didn't work.

https://wiki.libsdl.org/SDL2/SDL_HINT_VIDEO_MINIMIZE_ON_FOCUS_LOSS

### Screenshots or videos

https://github.com/ppy/osu/assets/64481039/328ef2d5-d127-4015-aa91-372a8dea6d40

### Version

2023.717.0

### Logs

[database.log](https://github.com/ppy/osu/files/12137249/database.log)
[input.log](https://github.com/ppy/osu/files/12137250/input.log)
[network.log](https://github.com/ppy/osu/files/12137251/network.log)
[performance.log](https://github.com/ppy/osu/files/12137252/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/12137253/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/12137254/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/12137255/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/12137256/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/12137257/runtime.log)

**Comment by bdach** [comment]
Why report non-working SDL envvars to us? What are we supposed to do about it?

**Reply by taep96** [reply]
Shouldn't it work? Searching on Google says to use that. Is there a better solution? It's pretty annoying to need to Alt+Tab twice even after switching the the correct workspace

**Reply by integerrr** [reply, marked as answer]
The point @bdach made still stands - this sounds like an SDL problem so you should probably take it up with the SDL team rather than reporting here

**Reply by taep96** [reply]
I'm not familiar with how SDL or osu works so I thought it's an issue with osu. Tried editing and removing [this line](https://github.com/ppy/osu-framework/blob/master/osu.Framework/Platform/SDL2Window.cs#L233) and only changing the value to 0 worked so I opened an issue [here](https://github.com/libsdl-org/SDL/issues/8028)
